source/docs/07-tablature.xml

```diff
@@ -96,50 +103,79 @@
+               <p>The <att>lines</att> attribute on the <gi scheme="MEI">staffDef</gi> element is
+                  used to define the number of lines, courses, or strings, present in the tablature.
+                  The <att>tab.courses</att> attribute is then used to enumerate the pitches of the
+                  open strings. It is important to note that this is given using the written pitch,
+                  not the sounding pitch. For example, the Western 6-string guitar, in standard
+                  tuning, sounds an octave below written pitch.</p>
+               <p>The <att>tab.courses</att> attribute gives the string tuning, ordered from highest
+                  to lowest pitch.</p>
+               <p>For standard guitar tuning, the <gi scheme="MEI">staffDef</gi> element might look
+                  like this:</p>
                <p>
                   <figure>
-                     <head/>
-                     <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample353.txt" parse="text"/></egXML>
+                     <head />
+                     <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code"
+                        xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample353.txt" parse="text"/></egXML>
                   </figure>
                </p>
-               <p>Chromatic alteration of the open string’s pitch may be indicated with the '-' or 'f' (flat), or the '#' or 's' (sharp). Multiple sharps and flats are not permitted.</p>
+               <p>Chromatic alteration of the open string’s pitch may be indicated with the '-' or
+                  'f' (flat), or the '#' or 's' (sharp). Multiple sharps and flats are not
+                  permitted.</p>
                <p>A guitar in E-flat tuning might look like this:</p>
                <p>
                   <figure>
-                     <head/>
-                     <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample354.txt" parse="text"/></egXML>
+                     <head />
+                     <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code"
+                        xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample354.txt" parse="text"/></egXML>
                   </figure>
                </p>
-               <p>Some instruments, like the 12-string guitar, have the four lowest strings tuned an octave above but are still written on a 6-line tablature staff. In this case, you may enumerate the open string pitches while maintaining 6 lines.</p>
+               <p>Some instruments, like the 12-string guitar, have the four lowest courses tuned in
```

Comment:
How do you tell where in the `courses` the `lines` values align? Can the courses possibly be grouped into the number of lines to make this explicit?

Reply:
This should be clearer now (partly because of the changes in how tuning is specified since this was written).

Reply:
@ahankinson can this be resolved?